exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -291,6 +302,11 @@ public synchronized void close() {
 
     logger.info("Shutdown completed ({} ms).", w.elapsed(TimeUnit.MILLISECONDS) );
     stateManager.setState(DrillbitState.SHUTDOWN);
+    // Interrupt the polling for shutdown since shutdown is triggered from WebUI.
```

Comment:
`Interrupt GracefulShutdownThread since Drillbit close is not called from it`

Reply:
Done.